data/Makefile.am

```diff
@@ -3,6 +3,7 @@ EXTRA_DIST=              \
         README           \
         doom.ico         \
         doom8.ico        \
+        doom.jpg         \
```

Comment:
This file can't be accepted since it contains copyrighted material.

Reply:
Fair enough. I'll ask around and see if we can get some original artwork for the Switch icons.

Reply:
Chocolate Doom logo?

This somewhat comes back to my question in the main reply. It's certainly worth upstreaming some of the more general portability fixes, but maybe the Switch stuff belongs more naturally in a fork.